i18n/ja/docusaurus-plugin-content-docs/current/guides/mobile-plugin-ima-android.md

```diff
@@ -29,9 +29,13 @@ This plugin is in the following open-source GitHub repository:
 
 To run this plugin, install the following:
 
-- [Google IMA SDK v3.30.3](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side)
-- [UID2 SDK for Android v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk) (for documentation, see [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
-- [UID2 IMA Plugin for Android v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk-ima)
+1. Google IMA SDK v3.30.3:
+   - [SDK](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side)
+   - [Release history](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side/history)
+1. UID2 SDK for Android v0.5.0:
+   - [SDK](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk)
+   - [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md)
```

Comment:
Not needed.  See comment in `docs/guides/mobile-plugin-gma-android.md`.

Reply:
As previously commented.

Reply:
Resolved above.